src/modules/commander/Commander.hpp

```diff
@@ -98,6 +98,17 @@ class Commander : public ModuleBase<Commander>, public ModuleParams
 private:
 
 	DEFINE_PARAMETERS(
+
+		(ParamInt<px4::params::NAV_DLL_ACT>) _datalink_loss_action,
+		(ParamInt<px4::params::COM_DL_LOSS_T>) _datalink_loss_threshold,
+		(ParamInt<px4::params::COM_DL_REG_T>) _datalink_regain_threshold,
+
+		(ParamInt<px4::params::COM_HLDL_LOSS_T>) _high_latency_datalink_loss_threshold,
```

Comment:
so for an `_obc_datalink_loss`, we need to add new params right?

Reply:
One more time for clarity, how would you like it to work?

Reply:
For example: I have a GCS link and a OBC link. I want the OBC link, if failing, to trigger a failsafe, but if the GCS link fails, I just want it to send a mavlink_log warning me, but not triggering a failsafe. Or, I may want them to both trigger a failsafe, or want that none trigger a failsafe but at least warn me this also applies to the high latency case.

Imagine we have 3 links. I would like to state which ones do trigger failsafes in the case of failure, but want to all of them to tell at least warn me in the case of a failsafe.

Reply:
It can make sense to be more atomic and say: OBC triggers a RTL failsafe, while GCS triggers a land failsafe

Reply:
TODO: we need to be careful for the scenario where a single mavlink connection will have multiple HEARTBEATs (eg companion also routing to GCS). I'll make sure that each received HEARTBEAT results in a telemetry_status publication.